packages/devtools_app/test/shared/device_dialog_test.dart

```diff
@@ -88,6 +121,89 @@ void main() {
       expect(find.text('CPU / OS: '), findsOneWidget);
       expect(find.text('Flutter Version: '), findsOneWidget);
       expect(find.text('Framework / Engine: '), findsOneWidget);
+      expect(find.text('Connected app type: '), findsOneWidget);
+      expect(find.text('Flutter native (debug build)'), findsOneWidget);
+      expect(find.text('VM Service Connection: '), findsOneWidget);
+    });
+
+    testWidgetsWithWindowSize(
+        'builds dialog for flutter native app (profile)', windowSize,
+        (WidgetTester tester) async {
+      final app = fakeServiceManager.connectedApp!;
+      mockConnectedApp(
+        app,
+        isFlutterApp: true,
+        isProfileBuild: true,
+        isWebApp: false,
+      );
+
+      deviceDialog = DeviceDialog(
+        connectedApp: app,
+      );
+
+      await tester.pumpWidget(wrap(deviceDialog));
+      expect(find.text('Device Info'), findsOneWidget);
+
+      expect(find.text('Dart Version: '), findsOneWidget);
```

Comment:
It feels a bit weird that we check for the line, but not necessarily that we are presenting the information that the feature is based around.
Would it be a big lift to check that we are presenting those values?

Reply:
Fixed 👍